examples/README.md

```diff
@@ -329,169 +334,153 @@ CC_EP="OR('SampleOrg.member')"
+Next we will write the code for the `run_test` function.
+Like in the case of Fabric, you install the FPC Chaincode using the `peer lifecycle` commands and then invoke transactions. 
+To install the FPC Chaincode, you need to use `$FPC_PATH/fabric/bin/peer.sh`.  This is a custom FPC wrapper to be used _instead_ of the `peer` cli command from Fabric.  `${PEER_CMD}` is set in `$FPC_PATH/fabric/bin/lib/common_ledger.sh` and conveniently points to the required script file.
 With the variables set and `common_ledger.sh` executed, usage of `peer.sh` is as follows:
 ```bash
-    try ${PEER_CMD} lifecycle chaincode package --lang fpc-c --label ${CC_ID} --path ${CC_PATH} ${PKG}
-    try ${PEER_CMD} lifecycle chaincode install ${PKG}
+${PEER_CMD} lifecycle chaincode package --lang fpc-c --label ${CC_ID} --path ${CC_PATH} ${PKG}
+${PEER_CMD} lifecycle chaincode install ${PKG}
 ```
 
-In the next step, the FPC chaincode must be approved by the organizations on the channel by agreeing on the chaincode definition.
+In the next step, the FPC Chaincode must be approved by the organizations on the channel by agreeing on the chaincode definition.
 ```bash
-    try ${PEER_CMD} lifecycle chaincode approveformyorg -o ${ORDERER_ADDR} -C ${CHAN_ID} --package-id ${PKG_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
-    try ${PEER_CMD} lifecycle chaincode checkcommitreadiness -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
-    try ${PEER_CMD} lifecycle chaincode commit -o ${ORDERER_ADDR} -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
+${PEER_CMD} lifecycle chaincode approveformyorg -o ${ORDERER_ADDR} -C ${CHAN_ID} --package-id ${PKG_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
+${PEER_CMD} lifecycle chaincode checkcommitreadiness -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
+${PEER_CMD} lifecycle chaincode commit -o ${ORDERER_ADDR} -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
 ```
 
 To complete the installation, we need to create an enclave that runs the FPC Chaincode.
 ```bash
-    # create an FPC Chaincode enclave
-    try ${PEER_CMD} lifecycle chaincode initEnclave -o ${ORDERER_ADDR} --peerAddresses "localhost:7051" --name ${CC_ID}
+# create an FPC Chaincode enclave
+${PEER_CMD} lifecycle chaincode initEnclave -o ${ORDERER_ADDR} --peerAddresses "localhost:7051" --name ${CC_ID}
 ```
 
-Add the following content to the function, `helloworld_test()` in test.sh.  Please note the inline comments for each of the commands.
-
+Add the following content to the function, `helloworld_test()` in `test.sh`.  Please note the inline comments for each of the commands.
 ```bash
-helloworld_test() {
-    say "- do hello world"
-
-    # install helloworld  chaincode
+run_test() {
```

Comment:
FPC chaincode .... chaincode as such is not a proper name so it shouldn't be a capital letter ... it is already is caps in the C of FPC ... so in theory we are repeating a word ....